static/src/js/components/Tour/SearchTour.scss

```diff
@@ -0,0 +1,157 @@
+:root {
+  --color-white: #ffffff;
+  --color-black: #000000;
+  --color-heading: #0063A6;
+  --color-gray: #9a989a;
+  --color-light-gray: #f5f5f5;
+  --font-mono: "DM Mono", SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace;
+}
+
+.tour {
+  &-heading {
+    font-family: var(--font-mono);
+    color: var(--color-heading);
+    font-size: 1.5rem;
```

Comment:
This is quite a bit larger than in the mockup. It should be 12px at the base font size, which would be `0.75rem`. Please do a quick check and make sure we are sticking to the font sizes in the figma.